src/Main.java

```diff
@@ -0,0 +1,161 @@
+import java.math.BigDecimal;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Main
+{
+    Pattern pattern = Pattern.compile("\\d{6}-\\d{2}-\\d{6}");
+    public int showMenu(Scanner s)
```

Comment:
```suggestion
    private int showMenu(Scanner s)
```
외부에 드러내도 되는 메소드인가, 외부에서 사용될 메소드인가를 고민하셔서 접근제어자를 설정해야합니다.